### ISSUE TITLE
Nextcloud flatpak can be opened multiple times

### ISSUE THREAD

**Issue by Firstyear** [flathub/org.nextcloud.Nextcloud#10]
It's possible to open multiple instances of the nextcloud flatpak, and all of them attempt to run and sync content at the same time. This may or may not cause consistency issues, so it would be great if the nextcloud flatpak was limited to a single instance running. 

Thanks,

**Comment by TingPing** [comment]
Does `flatpak run --filesystem=/tmp org.nextcloud.Nextcloud` change anything?

**Comment by juliushaertl** [comment]
I thought this was fixed with #6 but unfortunately it seems to work only every second time when running Nextcloud. So maybe this is an upstream bug.

**Comment by Firstyear** [comment]
Yes! Setting --filesystem=/tmp results in the second instance displaying:

```
Gtk-Message: Failed to load module "canberra-gtk-module"
Gtk-Message: Failed to load module "canberra-gtk-module"
Sys Info size:  0
int main(int, char**) Already running, exiting...
```

**Comment by juliushaertl** [comment]
@Firstyear does it also work if you try to open an instance for the third time?

**Comment by Firstyear** [comment]
Okay, it opens again on the third time ...

**Comment by juliushaertl** [comment]
I'm still not sure if this is an upstream issue or not. When running the client without flatpak it works as expected every time.

**Comment by prog-amateur** [comment]
Hello, did you find a solution ? 
I am on Ubuntu Wayland. Any flatpak app I have installed can open, while Nextcloud can't. 
Today, I have sync issues, and I can't simply open the UI. 
This issue was open on January, do you care about flatpak support ?

**Comment by juliushaertl** [comment]
I still could not find out why this is happening, seems like an upstream issue as the lock file is there and can be accessed by the client but it only recognizes it sometimes. However I cannot reproduce this outside of flatpak. If anyone has some ideas how this could be debugged, help is very welcome.

**Comment by Tids** [comment]
Could we set another lock file for the flatpak itself? In that way we can just ignore the nextcloud lock file.

**Comment by cwmke** [comment]
[For Debugging](http://docs.flatpak.org/en/latest/debugging.html)

**Comment by jurf** [comment]
Setting `--env=TMPDIR=/var/tmp` should be enough to get this to work…

**Comment by RobertBuhren** [comment]
Launching nextcloud with:

```
/usr/bin/flatpak run --branch=stable --arch=x86_64 --env=TMPDIR=/var/tmp  --command=nextcloud org.nextcloud.Nextcloud
```

Still allows multiple instances to run. Imho this makes the flatpak nextcloud app unusable,
